_docs/node-operator-requirements-service.md

```diff
@@ -0,0 +1,104 @@
+- Image Version: double check that this is in fact the latest node version to deploy. You can run into issues if you choose an older image for the node. Additionally, do a cross-reference with our [Github release page](https://github.com/OAK-Foundation/OAK-blockchain/releases) as well as Discord.
+- Collator: Make sure you select "Collator" so that your node can produce blocks once on-boarded. If you accidentally choose archive or full node, you'll have to go through the setup process again, as it is not easy to convert from one type of node to the other.
+- Cloud provider & region: We recommend changing up deployment locations and cloud providers. However, certain configurations can allow for a faster node sync time with OnFinality's [Lighting Restore](https://documentation.onfinality.io/support/Lightning-Restore.1651703824.html) feature.
+- Node & storage size: We recommend sticking with the recommended size based on the public network spec.
+
+![onfinality-setup-3](../assets/img/node-operator/onfinality-setup-3.png)
+
+*Step 3.3: Launch Configuration*
+The parachain and relay chain arguments should be up to date. However, it's always prudent to double check and cross-reference the respective chain specs. See relevant network sections above.
+
+![onfinality-setup-4](../assets/img/node-operator/onfinality-setup-4.png)
+
+*Step 3.4: Confirmation*
+Double check all of the parameters, configuration and costs. Once you're ready, click "Deploy Node".
+
+![onfinality-setup-5](../assets/img/node-operator/onfinality-setup-5.png)
+
+### Step 4: Sync and observe your node
+Once your node is successfully deployed, you can find it in the "Dedicated Nodes" section of the site. Click on your node and observe the following:
+- Parachain blocks are syncing appropriately
+- Relay chain blocks are syncing appropriately
+- No configuration errors in the console log. You can find your console by clicking on "Console logs" on the top right hand. 
+- CPU, Memory, and Storage are within reasonable range of use.
+
+Wait and observe in this step until your node is fully synced to the relay chain and parachain. It's important that BOTH are synced. You can cross-reference the respective network blocks on the [PolkadotJS App](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/explorer).
+
+![onfinality-setup-6](../assets/img/node-operator/onfinality-setup-6.png)
+
+### Step 5: Rotate keys in your collator node
+Once your node is fully synced, the next step is to setup an association with the node and its peers. To do so you need to generate a session key. The steps are as follows.
+*Step 5.1: Grab the WSS address* 
+In the monitoring view of your node, click the "API Endpoints" button on the top right. You'll see a number of options. Stick with "RPC", and copy the "Websocket" address, starting with `wss://node-*`.
+
+![onfinality-setup-7](../assets/img/node-operator/onfinality-setup-7.png)
+*Step 5.2: Navigate to PolkadotJS' RPC call* 
```

Comment:
Should add a new line above this. It looks odd when rendered.
![image](https://user-images.githubusercontent.com/16072671/168699543-3da4e16d-8692-46f3-928b-e5f5e3a421d4.png)